distribution/src/bin/elasticsearch.bat

```diff
@@ -68,6 +73,26 @@ IF "%checkpassword%"=="Y" (
   )
 )
 
+IF "%attemptautoconfig%"=="Y" (
```

Comment:
Similar to the discussion happening in https://github.com/elastic/elasticsearch/pull/77718#discussion_r710537307, we need to find a way to call to ConfigInitialNode with the following constraints: 

- Pass keystore password as stdin
- Not pollute classpath
- Not trigger  "multiple garbage collectors defined"  errors
- We don't (want to) have a CLI tool that we can use instead